### PR TITLE
Removes a bunch of organ booleans, organ flag cleanup.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
As per title.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

File: code/modules/mob/living/carbon/human/human_organs.dm [hunk 1]
```diff
@@ -137,7 +137,7 @@
 		return
 
 	for (var/obj/item/organ/external/E in organs)
-		if(!E || !E.can_grasp || (E.status & ORGAN_SPLINTED))
+		if(!E || (E.limb_flags & ORGAN_CAN_GRASP) || (E.status & ORGAN_SPLINTED))
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/human_organs.dm]: Initial check reads: `IF NOT E.can_grasp`. New check reads: `IF E.limb_flags HAS FLAG ORGAN_CAN_GRASP`. Looks like you accidentally inverted it? Unless the `can_grasp` boolean was inverted already.

File: code/modules/organs/organ_external.dm [hunk 2]
```diff
@@ -1065,7 +1062,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 
 /obj/item/organ/external/mechassist()
 	..()
-	cannot_break = 0
+	limb_flags &= ~ORGAN_CAN_BREAK
```
Comment (skull132) [inline code/modules/organs/organ_external.dm]:
```suggestion
	limb_flags |= ORGAN_CAN_BREAK
```

`cannot_break = 0` means CAN break.

File: code/modules/organs/subtypes/diona.dm [hunk 3]
```diff
@@ -64,8 +63,7 @@
 	w_class = 3
 	body_part = ARM_LEFT
 	parent_organ = BP_CHEST
-	can_grasp = 1
-	cannot_break = 1
+	limb_flags = ORGAN_CAN_MAIM | ORGAN_CAN_AMPUTATE
```
Comment (skull132) [inline code/modules/organs/subtypes/diona.dm]: Check left side. Missing `ORGAN_CAN_GRASP` flag?

File: code/modules/organs/subtypes/diona.dm [hunk 4]
```diff
@@ -90,8 +88,7 @@
 	body_part = LEG_LEFT
 	icon_position = LEFT
 	parent_organ = BP_GROIN
-	can_stand = 1
-	cannot_break = 1
+	limb_flags = ORGAN_CAN_MAIM | ORGAN_CAN_AMPUTATE
```
Comment (skull132) [inline code/modules/organs/subtypes/diona.dm]: Missing `ORGAN_CAN_STAND` flag?

File: code/modules/mob/living/carbon/human/human_organs.dm (hunk 1 above)
Comment (NonQueueingMatt) [reply code/modules/mob/living/carbon/human/human_organs.dm]: fixed

**Pull request merged**